packages/cms-cli/commands/auth.js

```diff
@@ -39,7 +34,7 @@ const {
 } = require('../lib/prompts');
 const { commaSeparatedValues } = require('../lib/text');
 
-const COMMAND_NAME = 'auth';
+const COMMAND_NAME = 'auth <type>';
```

Comment:
Here and throughout I cleaned up and normalized all of these.  Removed the `demand` option where relevant and instead put it here in the command name. SHould be consistent throughout now

Reply:
Also, I have this as <type> instead of [type] since it has a default... not sure if that is technically correct or not but it seems to work well